Dockerfile

```
@@ -1,4 +1,4 @@
-FROM ubuntu:22.04 as tomcat
+FROM lapierre/java-alpine:11 as tomcat
```

why not use a Docker vetted "Official image"? instead of pushing your own JDK version (having a full JDK is total overkill)
```suggestion
FROM eclipse-temurin:11-jre-alpine as tomcat
```